scripts/runtest.sh

```diff
@@ -92,13 +97,13 @@ function acceptance_test {
     if [ -n "$VERBOSE" ]
     then
         echo "# check for config file"
-        echo "TF_ACC=1 go test -tags '$tags' -v -timeout 120m ."
```

Comment:
This PR reduced the timeout from 120m to 90m. @vbauzysvmware please confirm this is OK as you increased it just recently.

Reply:
fixed